.storybook-react/webpack.config.js

```diff
@@ -0,0 +1,39 @@
+const path = require("path");
+const globImporter = require("node-sass-glob-importer");
+const packageConfig = require("../webpack.config.js");
+const MiniCssExtractPlugin = require("mini-css-extract-plugin");
+
+// Export a function. Accept the base config as the only param.
+module.exports = async ({ config, mode }) => {
+    // `mode` has a value of 'DEVELOPMENT' or 'PRODUCTION'
+    // You can change the configuration based on that.
+    // 'PRODUCTION' is used when building the static version of storybook.
+
+    //Add plugins
+    config.plugins.push(
+        new MiniCssExtractPlugin({
+            filename: "styles.css",
+            chunkFilename: "[id].css",
+        })
+    );
+
+    //Remove SVG config from storybook
```

Comment:
Can you explain this config to me? Currently I can't make heads or tails of it

Reply:
Storybook comes with a webpack config that it uses.  This file is transforming that config into a config that actually displays our stuff.  
So this line is getting rid of the default storybook svg config and L21 is telling it to treat SVGs the way we want to.